mediathread/templates/main/collection_add.html

```diff
@@ -39,14 +41,57 @@ <h2>Upload Media</h2>
                         <p class="lead">
                             Contribute video and audio to this course directly from your desktop into Mediathread with Wardenclyffe.
                         </p>
-                        <h3>Video</h3>
-
-
-
-                        <h3>Audio</h3>
-
-
 
+                        {% if can_upload and uploader %}
```

Comment:
The layout for the Upload section is a bit wonky...I think there are just some mismatched divs.